tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test for vagrant and packaged addons
+# Maintainer: dancermak <dcermak@suse.com>
+
+use strict;
+use warnings;
+use base "basetest";
+use testapi;
+use utils;
+use File::Basename;
+
+
+sub run() {
+    select_console('root-console');
+    zypper_call('in ansible');
+
+    select_console('user-console');
+
+    # version = Tumbleweed, Leap 15 etc
+    my $version = get_required_var('VERSION');
```

Comment:
To support Leap, you'd special case the `!is_tumbleweed` case and prepend `Leap-`.

Reply:
The changes in `lib/main_common.pm` should take care of that.